dom.bs

```diff
@@ -1217,7 +1217,32 @@ for discussion).
    <a>invoke</a> algorithm.
 
    <li><p>If <var>event</var>'s {{Event/type}} attribute value is not <var>listener</var>'s
-   <b>type</b>, terminate these substeps (and run them for the next <a>event listener</a>).
+   <b>type</b>, check for a legacy type attribute match by running these substeps:
+
+   <ol>
+    <li><p>If <var>event</var>'s {{Event/type}} attribute value is "animationend":
+    <ol>
+     <li><p>Let <var>original event type</var> be "animationend".
+     <li><p>Change the <var>event</var>'s {{Event/type}} attribute value to "webkitanimationend".
+    </ol>
+    <li><p>If <var>event</var>'s {{Event/type}} attribute value is "animationiteration":
+    <ol>
+     <li><p>Let <var>original event type</var> be "animationiteration".
+     <li><p>Change the <var>event</var>'s {{Event/type}} attribute value to "webkitanimationiteration".
+    </ol>
+    <li><p>If <var>event</var>'s {{Event/type}} attribute value is "animationstart":
```

Comment:
If the `<li>` has multiple children they need to start on their own line and be indented with a space. (Also applies earlier and later. Put the comment here for some breathing room.)